packages/http-client-java/emitter/src/code-model-builder.ts

```diff
@@ -244,6 +241,21 @@ export class CodeModelBuilder {
       versioning: { previewStringRegex: /$/ },
     }); // include all versions and do the filter by ourselves
 
+    // java namespace
+    if (this.options.namespace) {
```

Comment:
So the existing released libs will all use legacy mode?

Reply:
Yes.

Currently it is only our tests here (where most does not have `--namespace`, except for some that having invalid package e.g. `enum`).
And then there be unbranded, where we don't have any SDK in maintain.